### PR TITLE
added join command under basic

### LINKED ISSUE #233: create a reference page for 'join'

### PR DESCRIPTION
**Pull request by prenner**
# Advanced HW 7 submission

Closes #233

Name: Patrick Renner
Uniqname:prenner

### CODE REVIEW

File: _commands/basics/join.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+$ cat id.txt
+00,bob12
+01,john234
+02,carl44
+$ join -t, -o 1.1,2.1 -1 3 -2 1 scores.txt id.txt
+Bob,00
+John,01
+Carl,02
+~~~
+##### Break it down
```
Comment (mterwill) [inline _commands/basics/join.md]: I think you might need to add a newline in between 80 and 81 to get the heading to render correctly:

image

File: _commands/basics/join.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+$ join -1 1 -2 1 a.txt b.txt
+1 h i
+2 t o
+3 e h
+4 a h
+~~~
+
+##### Break it down
+
+ * The `-1` and `-2` option tells `join` to combine the file based on a specific column in the two files. In this case, join is combining the two files based on column 1 in a.txt (based on -1 flag) and column 1 in b.txt (based on -2 flag).
```
Comment (mterwill) [inline _commands/basics/join.md]: You use \`-1\` and \`-2\` in the first sentence, but not in the second.

File: _commands/basics/join.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mterwill) [inline _commands/basics/join.md]: No need for all these newlines!

**Comment by prenner** [comment]
Fixed those changes - Thanks!

**Comment by mterwill** [comment]
Looks great! Merging + giving you credit.

**Pull request merged**